tests/test_make_ballots.py

```diff
@@ -0,0 +1,8 @@
+from electos.ballotmaker import make_ballots
+
```

Comment:
Same package, so remove empty line.

Reply:
I installed `isort` to handle this stuff, but it's not running on save. I had to edit my VSCode settings.json to restore that behavior.

Reply:
Looks like isort is working on save!